### PR TITLE
WIP: Remove AWS credentials from github workflows

### PR DESCRIPTION
**Pull request by nopcoder**

### CODE REVIEW

File: .github/workflows/publish-spark-metadata-client.yaml [hunk 1]
```diff
@@ -37,14 +46,12 @@ jobs:
           password=${OSSRH_TOKEN}
           EOS
         env:
-          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           OSSRH_USERNAME: ${{ secrets.OSSRH_USERNAME }}
           OSSRH_TOKEN: ${{ secrets.OSSRH_TOKEN }}
+
       - name: Publish artifacts
```
Comment (itaiad200) [inline .github/workflows/publish-spark-metadata-client.yaml:52]: Did you check this one?

**Review by itaiad200 — CHANGES_REQUESTED** [review]
Thanks! Couple of questions:
1. Were the publishing flows tested?
2. What about other workflows that don't appear to be here? e.g. `goreleaser`

File: .github/workflows/publish-spark-metadata-client.yaml (hunk 1 above)
Comment (nopcoder) [reply .github/workflows/publish-spark-metadata-client.yaml:52]: Not yet - from the docs it supposes to use the machine credentials, and we assumed role.

**Comment by nopcoder** [comment]
@itaiad200 
Thanks, will mark this pr as draft - still work in progress.
The goreleaser changes already in master.

**Comment by nopcoder** [comment]
Too old - closing this one and probably will need to re-write to address the above.

**Pull request closed without merging**